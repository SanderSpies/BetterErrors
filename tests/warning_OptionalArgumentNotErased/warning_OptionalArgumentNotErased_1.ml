let asd a ?b = "asd"

let () = print_endline (asd 1 ~b:1)
