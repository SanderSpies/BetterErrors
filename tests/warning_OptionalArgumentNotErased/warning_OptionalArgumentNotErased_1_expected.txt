File "tests/warning_OptionalArgumentNotErased/warning_OptionalArgumentNotErased_1.ml", line 1, characters 11-12:
Warning 16: this optional argument cannot be erased.
